test/test_torch.py

```diff
@@ -11246,9 +11242,18 @@ def test_bitwise_ops(self):
         x_clone ^= y
         self.assertEqual(x_clone, xor_result)
 
-    def test_invert(self):
-        x = torch.ByteTensor([0, 1, 1])
-        self.assertEqual((~x).tolist(), [1, 0, 0])
+    def test_op_invert(self):
+        for t in (torch.ByteTensor, torch.ShortTensor, torch.IntTensor, torch.LongTensor):
```

Comment:
It would be better to make this iterate over the dtypes (e.g. `(torch.byte, torch.short, torch.int, torch.long)`)

Reply:
Makes sense.